src/ui/chat-window.js

```
@@ -90,7 +90,18 @@ export class ChatScene extends Phaser.Scene {
         this.show(false);
     }
 
-    // Create a row of text on the scroll window
+    // Takes in text and refreshes the display
+    writeText(text, format = FONTS.ITEM_HEADER) {
+        this.show();
```

I'm not sure I like force showing the log on every update... but I think the reason is that it hides part of the monster hitbox. I think ultimately it should show by default, but if the user hides the log then it shouldn't pop up on every update.

Also should probably just move the monsters up a bit so that having the log out doesn't make it annoying

---

Also if the log is hiding and there is an update written to it, the show/hide log button gets highlighted. That would be ideal here as well

---

Alternatively if you click on the enemy it could close the log

---

I think the only thing that should show/hide the log is the one button in the bottom left.

---

Alright I'll probably look into making some sort of indication that it's been updated without popping it up

---

Done, see updated screenshot in the description